docrepr/webview.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*
+
+"""
+View object documentation HTML in a separate GUI window.
+
+Requires PyQt4 or PySide. Original code by
+anatoly techtonik <techtonik@gmail.com> is placed in
+public domain.
```

Comment:
You copy/paste the whole working example and then strip all the meat and dependencies. But as I said - it is submitted into public domain, so after you merge it, you can strip all these "copyright" notes without any legal problems. I am okay with that.